ifrs17/Constants/Validations.ipynb

```diff
@@ -122,6 +123,7 @@
                 "\n    (Error.AocTypeCompulsoryNotFound      , _) => $\"Not all compulsory AoC Types have been imported.\",",
                 "\n    (Error.AocTypePositionNotSupported    , 1) => $\"The position of the AoC Type {s[0]} is not supported.\",",
                 "\n    (Error.AocConfigurationOrderNotUnique , _) => $\"Two or more AoC Configurations have the same Order.\",",
+                "\n    (Error.AccidentYearTypeNotValid       , 1) => $\"The parsed AccidentYear type {s[0]} is invalid. Expected Accident Year input of type int.\",",
```

Comment:
The parsed AccidentYear type {s[0]} is invalid. => 
The parsed AccidentYear {s[0]} is invalid.